app/modules/asset_groups/models.py

```diff
@@ -539,7 +552,7 @@ def __repr__(self):
 
     @property
     def bulk_upload(self):
-        return 'uploadType' in self.request and self.request['uploadType'] == 'bulk'
+        return 'uploadType' in self.config and self.config['uploadType'] == 'bulk'
```

Comment:
not precisely sure where the self.request stupp crept in but it was never correct